sekoia_automation/scripts/action_runner.py

```diff
@@ -0,0 +1,208 @@
+                if len(node.args) > 1 and isinstance(node.args[1], ast.Str):
+                    # provided as positional arg
+                    docker_param = node.args[1].s
+
+                elif len(node.keywords) > 0:
+                    # provided as keyword arg
+                    docker_param = node.keywords[0].value.s
+
+                module_item_to_docker_param[action_class] = docker_param
+
+        return module_item_to_docker_param
+
+    def get_manifest_by_docker_param(self, prefix: str, docker_param: str) -> dict:
+        manifests = self.__module_path.glob(f"{prefix}*.json")
+        for manifest_path in manifests:
+            with open(manifest_path) as file:
+                manifest = json.load(file)
+
+            if manifest.get("docker_parameters") == docker_param:
+                return manifest
+
+        return {}
+
+    @staticmethod
+    def get_module_item_type(cls):
+        def __iter_all_parents(c):
+            result = []
+            parent_cls = c.__bases__
+
+            result.extend(parent_cls)
+            for parent in parent_cls:
+                result.extend(__iter_all_parents(parent))
+
+            return result
+
+        parents_labels = {item.__name__: item for item in __iter_all_parents(cls)}
+        if "Connector" in parents_labels:
+            return "Connector"
+
+        elif "Action" in parents_labels:
+            return "Action"
+
+        elif "Trigger" in parents_labels:
+            return "Trigger"
+
+        raise ValueError("Incorrect class")
+
+    def run(self, args: dict) -> dict | None:
+        cls_to_docker = self.get_docker_params_from_main_py()
+        docker_param = cls_to_docker[self.__class_name]
+        manifest: dict = self.get_manifest_by_docker_param(
+            docker_param=docker_param, prefix=""
+        )
+
+        arguments_schema = manifest.get("arguments")
+        results_schema = manifest.get(
+            "results", {}
+        )  # no `results` schema for connectors
+
+        # check inputs
+        validate(instance=args, schema=arguments_schema)
+
+        # find and load Module class
+        module_class_name, module_class_path = self.find_file_with_child_class(
+            parent_class_to_find="Module"
+        )
+
+        if module_class_name is None or module_class_path is None:
+            module = Module()
+
+        else:
+            module_cls = self.load_class_from_path(module_class_path, module_class_name)
+
+            # Prepare module configuration
+            module_annotations = module_cls.__annotations__
+            module_config_cls = module_annotations["configuration"]
+            conf_args = {key: args.get(key) for key in module_config_cls.__fields__}
+            module_conf = module_config_cls(**conf_args)
+
+            module = module_cls()
+            module.configuration = module_conf
+
+        file_path = self.find_file_with_module_item_class()
+        module_item_cls = self.load_class_from_path(
```

Comment:
I tested the script with an action from the Harfanglab automation module and got an error about the harfanglab python module:

```
$ cd Harfanglab
$ poetry install
$ poetry run pip install git+https://github.com/SEKOIA-IO/sekoia-automation-sdk.git@feat/add_action_test_script
$ poetry run sekoia-automation run-action --module-name=HarfangLab --class-name=GetProcessListAction --modules-path .. target_agents=cs-vm
...
moduleNotFoundError: No module named 'Harfanglab'
```